### PR TITLE
17772: STEM AD: TIMS Work

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description of change
As a developer, I need to create a new TIMS doc type and corresponding spool file to record a permanent record of the STEM automated decision.

This PR contains the change made in https://github.com/department-of-veterans-affairs/vets-api/pull/5608

## Original issue(s)
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/17772

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: app/workers/education_form/forms/va10203.rb [hunk 1]
```diff
@@ -2,8 +2,17 @@
...
 module EducationForm::Forms
   class VA10203 < Base
+    def initialize(app)
+      @education_stem_automated_decision = app.education_stem_automated_decision
+      super(app)
+    end
+
     def header_form_type
-      'V10203'
+      @education_stem_automated_decision&.decision_state == 'denied' ? '10203DNY' : 'V10203'
```
Comment (zurbergram) [inline app/workers/education_form/forms/va10203.rb:11]:
```suggestion
      @education_stem_automated_decision&.automated_decision_state == 'denied' ? '10203DNY' : 'V10203'
```

File: app/workers/education_form/forms/va10203.rb [hunk 2]
```diff
@@ -2,8 +2,17 @@
...
+    end
+
+    def form_identifier
+      @education_stem_automated_decision&.decision_state == 'denied' ? 'VA Form 22-10203DNY' : 'VA Form 22-10203'
```
Comment (zurbergram) [inline app/workers/education_form/forms/va10203.rb:15]:
```suggestion
      @education_stem_automated_decision&.automated_decision_state == 'denied' ? 'VA Form 22-10203DNY' : 'VA Form 22-10203'
```

**Review by zurbergram — CHANGES_REQUESTED** [review]
See schema for column names

**Review by sonntag-adam — APPROVED** [review]

**Pull request closed without merging**